packages/functional-tests/playwright.config.ts

```diff
@@ -73,6 +73,8 @@ export default defineConfig<PlaywrightTestConfig<TestOptions, WorkerOptions>>({
             ),
           },
         ],
+        ['blob'],
```

Comment:
Still not seeing the report.zip in the artifacts dir.

Not sure if we need to set the blob [`outputDir` property](https://playwright.dev/docs/test-reporters#blob-reporter) to `path.resolve(__dirname, '../../artifacts/tests')` roughly similar to how we set the custom JUnit `outputFile`.

Reply:
Yeah, I was thinking it would anyways generate the zip file somewhere but I am not seeing it. I will set the path and see if it does.

Reply:
Though when I tested it locally it generated a blob-report.zip file inside the functional-test dir.